lldb/source/Host/linux/Host.cpp

```diff
@@ -213,6 +213,11 @@ static bool GetStatusInfo(::pid_t Pid, ProcessInstanceInfo &ProcessInfo,
     } else if (Line.consume_front("Tgid:")) {
       Line = Line.ltrim();
       Line.consumeInteger(10, Tgid);
+    } else if (Line.consume_front("CoreDumping:")) {
+      uint32_t coredumping;
+      Line = Line.ltrim();
+      Line.consumeInteger(1, coredumping);
```

Comment:
This is calling:
```
    template <typename T> bool consumeInteger(unsigned Radix, T &Result) {
```
* Radix of 1 means binary?
* Parsing can fail, though now I look above and nothing else checks the return value, sigh...

This is new though, so can you make it check the return value and not set if it's false? The type is optional<bool>, so anyone making use of this is handling the "I don't know" state already.

Reply:
Good suggestion, I'll validate the return.